src/passes/Asyncify.cpp

```diff
@@ -1689,6 +1724,16 @@ struct Asyncify : public Pass {
       runner.setValidateGlobally(false);
       runner.run();
     }
+    if (asserts) {
+      // Add asserts in non-instrumented code. Note we do not use an
+      // instrumented pass runner here as we do want to run on all functions.
```

Comment:
What is an "instrumented pass runner"?

Reply:
A pass runner with some instrumentation to decide on which functions to run. Specifically we just want to run on functions that need the asyncify instrumentation. Maybe the word is used in two slightly different manners here? Can improve that in a followup if there are better ideas.